docs/DeveloperGuide.md

```diff
@@ -410,6 +504,40 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
   * 2a1. TAHub shows an error message stating that the index is invalid.
   * 2a2. Use case ends.
 
+**<u>Use case: UC6 - Create a Consultation</u>**
+
+**MSS**
+
+1. User enters: `addconsult d/2024-10-20 t/14:00`
+2. System validates date and time
+3. System creates consultation
+4. Success message shown
+
+**Extensions**
+
+* 1a. Invalid date/time format
+    * 1a1. System shows error message
+    * 1a2. Use case ends
+
+* 1b. Date/time already booked
+    * 1b1. System shows conflict error
+    * 1b2. Use case ends
+
+**<u>Use case: UC7 - Add Student to Consultation</u>**
+
+**MSS**
+
+1. TA lists existing consultations
```

Comment:
Can add what if theres no existing consultations/no consultations that are not over yet